modin/pandas/test/dataframe/test_default.py

```diff
@@ -643,7 +655,22 @@ def test_pivot(data, index, columns, values):
         pytest.param("nunique", id="full_axis_func"),
     ],
 )
-def test_pivot_table_data(data, index, columns, values, aggfunc):
+def test_pivot_table_data(data, index, columns, values, aggfunc, request):
+    if (
```

Comment:
It’s not pretty, but it’s difficult to select the desired condition through parameters. It is also possible that there are several problems that will need to be investigated separately.